src/main/kotlin/io/snyk/plugin/cli/Vulnerability.kt

```diff
@@ -42,13 +44,7 @@ class Vulnerability {
 
     // todo: (by Pavel)
     //  How about using comparator for tree nodes? Issue I see atm: this code cannot handle other issues e.g. critical if this will come later.
```

Comment:
that todo could probably be removed now as requested abstraction is now implemented in the next line? cc @pavel-snyk